sdk-dotnet/LittleHorse.Sdk.Tests/Resources/littlehorse.config

```diff
@@ -0,0 +1,5 @@
+LHW_TASK_WORKER_VERSION=test
```

Comment:
I don't think we want to include these temporary files to the test resources. @sauljabin what do you think about this?

Reply:
not need it, check this out: https://github.com/littlehorse-enterprises/littlehorse/blob/master/canary/src/test/java/io/littlehorse/canary/config/ConfigLoaderTest.java#L66

Yo can create temp files when executing tests, so my recommendation do not commit this files